include/Utils.hpp

```diff
@@ -1,128 +1,22 @@
 #pragma once
 
 #include "QeccException.hpp"
-#include "TreeNode.hpp"
+#include "gf2dense.hpp"
```

Comment:
```suggestion
#include <gf2dense.hpp>
```

As a third party include, this should use the `<...>` syntax.